README.md

```
@@ -1,15 +1,51 @@
+  * Softhsm client zip.
+  * i18n bundles for multiple MOSIP UI's.
+  * Masterdata templates for admin service module.
+  * themes for multiple UI's.
+* All these artifacts are released as a part of the Mosip Release with some of them being taken from open sourced repository as per the need.
 
 All these artifacts are released as a part of the Mosip Release with some of them being taken from open sourced repository as per the need.
+
+## Deployment in K8 cluster with other MOSIP services:
+### Pre-requisites
+* Set KUBECONFIG variable to point to existing K8 cluster kubeconfig file:
+  * ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)